test/conformance/ingressv2/util.go

```diff
@@ -308,7 +308,7 @@ func CreateTimeoutService(ctx context.Context, t *testing.T, clients *test.Clien
 		},
 	}
 
-	return name, port, createPodAndService(ctx, t, clients, pod, svc)
+	return name, gatewayv1alpha1.PortNumber(port), createPodAndService(ctx, t, clients, pod, svc)
```

Comment:
Same comment as in the other PR: Should we do this at the call-site? Is it valuable to keep these functions generic between our tests?

Reply:
Yes, I wondered this. Updated the util function.